modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/manager/v1_0/DefaultObjectEntryManagerImpl.java

```diff
@@ -1346,6 +1206,166 @@ else if ((objectField.getListTypeDefinitionId() != 0) &&
 		return values;
 	}
 
+	private void _upsertAndRelateNestedObjectEntry(
```

Comment:
For the following pull requests, maybe better to update first the name and then, the method inside